source/tutorials/packaging-existing-software.md

```diff
@@ -8,26 +8,42 @@ myst:
 (packaging-existing-software)=
 # Packaging existing software with Nix
 
-One of Nix's primary use-cases is in addressing common difficulties encountered while packaging software, like managing dependencies.
+One of Nix's primary use-cases is in addressing common difficulties encountered with compiling software, such as specifying and obtaining dependencies.
 
-In the long term, Nix helps tremendously in alleviating that stress, but when *first* packaging existing software with Nix, it's common to encounter missing dependencies preventing builds from succeeding.
+In the long term, Nix helps tremendously with alleviating such problems.
+But when *first* packaging existing software with Nix, it's common to encounter errors that seem inscrutable.
 
-In this tutorial, you'll create your first Nix derivations to package C/C++ software, taking advantage of the [Nixpkgs Standard Environment](https://nixos.org/manual/nixpkgs/stable/#part-stdenv) (`stdenv`) which automates much of the work of building self-contained C/C++ packages.
+## Introducton
 
-The tutorial begins by considering `hello`, an implementation of "hello world" which only requires dependencies provided by `stdenv`.
+In this tutorial, you'll create your first [Nix derivations](https://nix.dev/manual/nix/2.18/language/derivations) to package C/C++ software.
+This takes advantage of the [Nixpkgs Standard Environment](https://nixos.org/manual/nixpkgs/stable/#part-stdenv) (`stdenv`), which reliably automates much of the work involved.
 
+### What will you learn?
+
+The tutorial begins with `hello`, an implementation of "hello world" which only requires dependencies already provided by `stdenv`.
 Next, you will build more complex packages with their own dependencies, leading you to use additional derivation features.
 
 You'll encounter and address Nix error messages, build failures, and a host of other issues, developing your iterative debugging techniques along the way.
 
+### What do you need?
+
+- Familiarity with the Unix shell and plain text editors
+- You should be confident with [reading the Nix language](reading-nix-language). Feel free to go back and work through the tutorial first.
+
+### How long does it take?
+
+Going through all the steps carefully will take ca. 60 minutes.
+
+## Your first package
+
 :::{note}
-A _package_ is an informally defined Nixpkgs concept referring to a Nix derivation representing an installation of some project.
+A _package_ is a loosely defined Nixpkgs concept that refers to both a collection of files and other data, or a Nix derivation representing such a collection before it comes into being.
```

Comment:
```suggestion
A _package_ is a loosely defined Nixpkgs concept that refers to both a collection of files and other data, or a Nix derivation representing such a collection.
```

I am uncertain myself about this suggestion I am making. 
We don't want to imply that the derivation exists and matters only before the package realisation?

Reply:
It didn't say *only*. That's exactly why derivations matter, they're the "before package artifact exists" part. But I don't think we need to go into that depth here.